src/components/WorkChart.astro

```diff
@@ -0,0 +1,17 @@
+---
+type Props = {
+  img: string;
+  chartModifier: string | null;
+  href: string | null;
```

Comment:
I understand why href could be used here, but in this context something like "linkTo" would be better PropName. We know it's gonna be in a link, but let's make it more "humans friendly"